### PR TITLE
docs(ngDblclick): describe your change...

### PR DESCRIPTION
**Pull request by aalpgiray**
[Review on Reviewable](https://reviewable.io/reviews/angular/angular.js/12238)

### CODE REVIEW

File: src/ng/directive/ngEventDirs.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ forEach(
  * @name ngDblclick
  *
  * @description
- * The `ngDblclick` directive allows you to specify custom behavior on a dblclick event.
+ * The `ngDblclick` directive allows you to specify custom behavior on a dblclick event. Unless used with "ng-click" then ng-click not fires.
```
Comment (Narretz) [inline src/ng/directive/ngEventDirs.js]: Sorry, I don't understand what this means. Which event does not fire when dblclick is used with click?

Comment (caitp) [inline src/ng/directive/ngEventDirs.js]: not really getting this either

**Comment by caitp** [comment]
@aalpgiray if you rewrite this more clearly, we could consider landing this. But currently this doesn't seem to mean anything

**Comment by Narretz** [comment]
I'm gonna close this because we haven't got an update from @aalpgiray Feel free to reopen with an uodate.

**Pull request closed without merging**